### PR TITLE
Bug 1131022 - Investigate mozIccManager is undefined test failures

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[mwargers Martijn (mwargers)](https://github.com/mwargers) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ee77bda74cd6)

File: tests/python/gaia-ui-tests/gaiatest/gaia_test.py [hunk 1]
```diff
@@ -198,22 +196,16 @@ def insert_contact(self, contact):
         self.marionette.set_context(self.marionette.CONTEXT_CONTENT)
 
     def insert_sim_contact(self, contact, contact_type='adn'):
-        # TODO Bug 1049489 - In future, simplify executing scripts from the chrome context
-        self.marionette.set_context(self.marionette.CONTEXT_CHROME)
```
Comment (davehunt) [inline tests/python/gaia-ui-tests/gaiatest/gaia_test.py]: Why do we not need to switch to the system app here?

File: tests/python/gaia-ui-tests/gaiatest/gaia_test.py [hunk 2]
```diff
@@ -198,22 +196,16 @@ def insert_contact(self, contact):
...
         mozcontact = contact.create_mozcontact()
         result = self.marionette.execute_async_script('return GaiaDataLayer.insertSIMContact("%s", %s);'
                                                       % (contact_type, json.dumps(mozcontact)), special_powers=True)
         assert result, 'Unable to insert SIM contact %s' % contact
-        self.marionette.set_context(self.marionette.CONTEXT_CONTENT)
         return result
 
     def delete_sim_contact(self, moz_contact_id, contact_type='adn'):
-        # TODO Bug 1049489 - In future, simplify executing scripts from the chrome context
-        self.marionette.set_context(self.marionette.CONTEXT_CHROME)
```
Comment (davehunt) [inline tests/python/gaia-ui-tests/gaiatest/gaia_test.py]: As above, why do we not need to switch to the system app here?

**Comment by try-server-hook** [comment]
[mwargers Martijn (mwargers)](https://github.com/mwargers) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e8cbb31af0e4)

**Pull request merged**